CMakeLists.txt.user

```diff
@@ -0,0 +1,540 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file shouldn't be in the PR.

Reply:
Oops, sorry about that. Should be added to .gitignore I guess.